_posts/2016-02-29-phishing.md

```diff
@@ -0,0 +1,54 @@
+
+### 5. Internet Content Filtering (ICF) and restricted outbound internet access.
+
+A further measure that can help is internet content filtering on outbound web requests from employee workstations and laptops. ICF is a mechanism that intercepts outbound web requests and verifies whether they are to be permitted. Very often a phishing email will not have a malicious payload itself, in order to improve its chances of being delivered to targeted victims. It will instead contain a seemingly innocuous word document or Excel document attachment with a simple macro that executes on file open and attempts to download the real virus payload, or just a link to a website. Usually this file or webpage will be hosted on known malicious domains and ICF can actively block access to these domains so that the payload cannot be downloaded or form that attempts to capture their password cannot be loaded.
+
+### 6. Host permissions/group policy and Data Execution Prevention
+
+Assuming that all the above measures have failed, modern operating systems such as Microsoft Windows 10 incorporate DEP or Data Execution Prevention and Group Policies that prevent untrusted executables or those attempting to access certain areas of memory from executing. A well configured group policy can block the execution of downloaded executables.
+
+### 7. Antivirus
+
+If a phishing email succeeds in bypassing all the above measures and manages to download and execute, all is still not lost. Modern antivirus tools can perform a range of checks on attachments, from checking them against known virus patterns, to loading a copy of an attachment into a sandboxed area of memory and executing them in order to check for malicious calls or activity, before allowing the user access to the real attachment themselves.
+
+### 8. Password management, expiry
+
+Finally, if the worst does happen, and a phishing email is received by a user and successfully tricks them into entering their password, all is not lost. By enforcing strong password management, you can ensure that users are using different passwords for each site or service they use, meaning that the damage in the credential loss is known and limited. By enforcing frequent password rotation/expiry you can also ensure that the window during which a malicious party can exploit any captured credentials is limited.
+
+The above example is not an exhaustive list of every measure that can be taken against phishing attacks, but does demonstrate that very often the strength of a security solution lies not at any single layer or component, but in a so-called defense in depth approach, in which a range of measures at different layers and types are stacked to provide a robust overarching protection against a given threat
```

Comment:
Maybe title this summing up paragraph "Defense in Depth" or similar so it is separated out from point number 8.